dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -13,18 +13,34 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
   # GET /users/auth/clever/callback
   def clever
     if should_connect_provider?
-      connect_provider
+      return connect_provider
```

Comment:
this could be cleaned up into a single line: `return connect_provider if should_connect_provider?`